packages/desktop-client/src/components/Settings.js

```diff
@@ -473,6 +473,37 @@ function SettingsLink({ to, name, style, first, last }) {
   );
 }
 
+function Version() {
+  let [version, setVersion] = useState('');
+
+  useEffect(async () => {
+    const { error, version } = await send('get-server-version');
+
+    if (error) {
+      setVersion('');
+    } else {
+      setVersion(version);
+    }
+  }, []);
+
+  return (
+    <Text
+      style={[
+        {
+          alignSelf: 'center',
+          color: colors.n7,
+          ':hover': { color: colors.n2 },
+          padding: '6px 10px'
+        },
+        styles.staticText,
+        styles.smallText
+      ]}
+    >
+      v{window.Actual.ACTUAL_VERSION} | {version ? `v${version}` : 'N/A'}
```

Comment:
any issues I might be unaware of with using the `window.Actual` object for the app version? There is a `get-version` handler that could be used instead.

Reply:
(Sorry for the delay)

Nope! This should be correct; I think the `get-version` is left around from older code when the frontend code didn't have direct access to it